hq/app/aws/iam/IAMClient.scala

```diff
@@ -39,5 +40,12 @@ object IAMClient {
     } yield report
   }
 
-}
-
+  def getAllCredentialReports(accounts: Seq[AwsAccount])(implicit executionContext: ExecutionContext): Seq[Attempt[(AwsAccount, CredentialReportDisplay)]] = {
```

Comment:
As discussed IRL, this should probably have a different signature. Off the top of my head something like the following, but compare it to the Security Groups implementation.

`Attempt[List[(AwsAccount, Either[FailedAttempt, CredentialReportDisplay])]]`

Reply:
I'll use `Attempt.successfulAttempts` which is accepting a parameter of `attempts: List[Attempt[A]]` so will ignore the failing attempts.

Reply:
done.